ja/lessons/basics/enum.md

```diff
@@ -28,15 +28,14 @@ at/3
 ...
 ```
 
-これを見れば`Enum`モジュールに大量の機能があるのが明らかで、これには明確な理由があります。
-列挙は関数型プログラミングの核で、信じられないほど有用です。
-おまけに列挙は、以前見たような言語レベルでサポートされているドキュメントといった、Elixirの他の要素と共に活用することで、信じられないような効果をもたらします。
+このように、`Enum`モジュールが非常に多くの機能を持っていることは一目瞭然ですが、これには明確な理由があります。
+列挙は関数型プログラミングの中核であり、Elixir が開発者にもたらす驚くべきその他の恩恵と基部で統合的に用意されているおかげです。
 
 全ての関数を知りたい場合は公式ドキュメントの[`Enum`](https://hexdocs.pm/elixir/Enum.html)を参照してください。尚、列挙の遅延処理では[`Stream`](https://hexdocs.pm/elixir/Stream.html)モジュールを利用してください。
 
 ### all?
 
-`all?`を使うとき、そして`Enum`の多くのケースで、コレクションの要素に適用する関数を渡します。`all?`の場合には、コレクション全体でこの関数は`true`と評価されなければならず、これを満たさない場合は`false`が返ります。
+通常、 `all?/2`にはコレクションの要素に対して適用する関数を渡します。`all?/2`の場合、コレクションの全体が`true`と評価されなければならず、そうでなければ`false`が返されます。
```

Comment:
Is deleted an `Enum` intentional?

Reply:
Yes, it is my intention.
I believe the original words are not always necessary for translation.